tests/test_ops/test_onnx.py

```diff
@@ -46,12 +46,19 @@ def test_nms():
             opset_version=11)
     onnx_model = onnx.load(onnx_file)
 
+    ort_custom_op_path = get_onnxruntime_op_path()
+    if not os.path.exists(ort_custom_op_path):
+        pytest.skip('softnms for onnxruntime is not compiled.')
```

Comment:
Please change the log from softnms to nms.

Reply:
ok